scripts/generate_skip_file.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+with open("data.txt", "r") as f:
+    data = f.read()
+
+matches_failed = re.findall(r"thread '(.*)' panicked at ", data)
+matches_failed = matches_failed[1:]
+
+matches_failed = [
+    (
+        m.split("::")[-2]
+        .replace("-", "_minus_")
+        .replace("+", "_plus_")
+        .replace("^", "_xor_"),
```

Comment:
should be the other way around for all of them

Reply:
fixed.